### DISCUSSION TITLE
selectCheckboxMenu Event:itemSelect is not supported.

### DISCUSSION THREAD

**Discussion by sc-nm** [primefaces/community#2734] [PrimeFaces]
Hi,

the ajax event="itemSelect" for a selectCheckboxMenu as shown [here ](https://www.primefaces.org/showcase/ui/input/checkboxMenu.xhtml?jfwid=bf936) does not seem not work in Primefaces 8.0.+:

`<p:ajax> Event:itemSelect is not supported`

But the event="itemUnselect" works fine. Is this a bug, or is there any other smart way to detect a selection of a single element in a checkbox menu? If so, where in the documentation is it hidden?

Best regards

**Comment by tandraschko** [comment]
Its there in the newest version. 8 is already many years old. Just upgrade.
